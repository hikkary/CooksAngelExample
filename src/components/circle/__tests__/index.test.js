
import React from 'react';
import Circle from '../';
import 'jest-styled-components';

import { shallow } from 'enzyme';


it('should match the snapshot', () => {
  const component = shallow(
    <Circle  />
  );

  expect(component).toMatchSnapshot();
});

