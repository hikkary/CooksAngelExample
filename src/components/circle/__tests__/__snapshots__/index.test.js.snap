// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  background-color: yellow;
  height: 50px;
  widht:50px;
  border-radius: 25px;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
